client/app/dedicatedCloud/translations/Messages_fr_FR.xml

```diff
@@ -822,7 +822,8 @@
    <translation id="dedicatedCloud_datacenter_secondary_datacenter_list" qtlid="520143">Liste des datacenters secondaires</translation>
    <translation id="dedicatedCloud_datacenter_secondary_datacenter_name_placeholder" qtlid="520156">Choisissez le datacenter secondaire</translation>
    <translation id="dedicatedCloud_datacenter_secondary_ip_address_pool_available" qtlid="520169">Pool d’adresses IP publiques disponible du Private Cloud secondaire</translation>
-   <translation id="dedicatedCloud_datacenter_order_ip_secondary_pcc" qtlid="520182">Vous n’avez aucun pool d’adresses IP disponible sur ce Private Cloud. Vous pouvez en commander</translation>
+   <translation id="dedicatedCloud_datacenter_order_ip_secondary_pcc_part_one">Vous n’avez aucun pool d’adresses IP disponible sur ce Private Cloud. Veuillez en </translation>
```

Comment:
Truncate your sentences in that way will result in a issue from the Translator point of view.